### PR TITLE
GMPE01, GP5E01, GP6E01, GP7E01, RM8E01: Code Updates

### PR DESCRIPTION
**Pull request by EndangeredNayla**

### CODE REVIEW

**Comment by Pokechu22** [comment]
@dolphin-emu-bot rebuild

**Comment by Pokechu22** [comment]
@dolphin-emu-bot rebuild

**Comment by AdmiralCurtiss** [comment]
Please explain what these updates are for.

**Comment by EndangeredNayla** [comment]
This pr addresses code changes made in the mario party netplay community..   codes were made to improve gameplay.

**Review by Rumi-Larry — COMMENTED** [review]
A bunch of names for cheats have abnormal grammar, that needs checking by a native English speaker.
Also, I recommend that all instances of "Last 5 Turns" be between parentheses and with the word "events" appended e.g. ["Last 5 Turns" events happen....]. Otherwise it's very confusing to read.

File: Data/Sys/GameSettings/GP7E01.ini [hunk 1]
```diff
@@ -1013,18 +1039,13 @@ $Mechanics - Hammer Bro Orb Steals 20 Coins [Ralf]
 *Hammer Bro steals 20 coins instead of 10 when landing on his space.
 
 $Mechanics - Happening Star Becomes Star Star [Airsola]
```
[the remark this answers is not in the document]
Comment (EndangeredNayla) [reply Data/Sys/GameSettings/GP7E01.ini:1041]: This is technically accurate, it is a cheat code that gives  a bonus star for someone who had the most stars at one point.

**Comment by Pokechu22** [comment]
@dolphin-emu-bot rebuild

**Comment by EndangeredNayla** [comment]
any chance for a review?

**Comment by Simonx22** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**